ibm/mas_devops/roles/suite_rollback/defaults/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+mas_rollback_dryrun: "{{ lookup('env', 'MAS_ROLLBACK_DRYRUN') | default('False', True) | bool }}"
+
+mas_core_version: "{{ lookup('env', 'MAS_CORE_VERSION') }}"
+rollback_mas_core: "{{ lookup('env', 'ROLLBACK_MAS_CORE') | default('True', True) | bool }}"
+verify_core_version: "{{ lookup('env', 'VERIFY_CORE_VERSION') | default('False', True) | bool }}"
+mas_instance_id: "{{ lookup('env', 'MAS_INSTANCE_ID') }}"
+mas_namespace: "mas-{{ mas_instance_id }}-core"
+
+skip_compatibility_check: "{{ lookup('env', 'SKIP_COMPATIBILITY_CHECK') | default('False', True) | bool }}"
```

Comment:
Should SKIP_COMPATIBILITY_CHECK be in the docs?

Reply:
SKIP_COMPATIBILITY_CHECK environment is meant only for development. It should be always 'False' by default, however, in dev mode, it is set to 'True'. If the user tries to rollback to any version which is not in supported list, it should fail with a proper message to the user. However, in the dev, we accept rollback from one pre built version to another on the same base version though it is not listed under supported versions. For ex: 8.11.0-pre.dev to 8.11.0-pre.stable.

Reply:
@sekharcvalluri Can you add a comment to this file just above skip_compatibility_check so that we know it is for dev reasons only and is not expected to be changed by an end user?

Reply:
Done